study-group/Toast-Implementation-Studies.md

```diff
@@ -74,21 +74,137 @@ The `restoreFocusOnClear` attribute restores page's focus once the toast is clea
+import Snackbar from '@material-ui/core/Snackbar';
+
+class SimpleSnackbar extends Component {
+  state = {
+    open: false,
+  };
+
+  handleClick = () => {
+    this.setState({ open: true });
+  };
+
+  handleClose = (event, reason) => {
+    this.setState({ open: false });
+  };
+
+  render() {
+    return (
+      <div>
+        <Button onClick={this.handleClick}>Open simple snackbar</Button>
+        <Snackbar
+          anchorOrigin={{
+            vertical: 'bottom',
+            horizontal: 'left',
+          }}
+          open={this.state.open}
+          autoHideDuration={6000}
+          onClose={this.handleClose}
+          message={<span id="message-id">Note archived</span>}
+          action={[
+            <Button key="undo" color="secondary" size="small" onClick={this.handleClose}>
+              UNDO
+            </Button>,
+            <Button key="close" color="secondary" size="small" onClick={this.handleClose}>
+              CLOSE
+            </Button>
+            ...
+          ]}
+        />
+      </div>
+    );
+  }
+}
+```
+
+The Snackbar is a React component, with the options passed to it as props.
+Styling the component can be done by passing to the `classes` property of a [SnackbarContent](#snackbarcontent) component (see link):
+
+```jsx
+<SnackbarContent
+  classes={{
+    root: {
+      'background-color': 'green'
+    },
+    message: {
+      color: 'blue',
+      'font-size': '24px'
+    },
+    action: {
+      border: '5px'
+    }
+  }}
+  ...
+/>
+```
+
+Per the [Material-UI overriding classes spec](https://material-ui.com/customization/overrides/#overriding-with-classes),
+the developer can provide styling on predefined parts of the component.
+
+Finally, the Material-UI library comes with helpful built-ins for creating components,
+including transitions like Fade:
+
+```jsx
+import Fade from '@material-ui/core/Fade';
+
+...
+
+<Snackbar
+  TransitionComponent={Fade}
+  ...
+/>
+```
+
+### Notable Features & Details
+
+#### SnackbarContent
+Material-UI provides both `<Snackbar>` and `<SnackbarContent>` components.
+The SnackbarContent component lets the developer style the `root`, `message`, and `action` via the `classes` property.
+The customized `<SnackbarContent>` view can then be used by adding it as a child of a regular `<Snackbar>`.
+The `classes` property of Snackbar lets the developer style the `root`,
+and each of the possible positions the toast can be in (`anchorOriginTopCenter`, `anchorOriginBottomCenter`, etc.).
+This is likely to allow finer control over the exact placement in each position.
+
+#### resumeHideDuration
+The Snackbar component has a property that allows the developer to reset the timeout to a specified number of milliseconds after user interaction.
+
+#### notistack
+It is not natively supported to display multiple Snackbars at a time, 
+but the spec recommends using the [notistack](https://github.com/iamhosseindhv/notistack) library to make it possible.
```

Comment:
```suggestion
but the documentatoin recommends using the [notistack](https://github.com/iamhosseindhv/notistack) library to make it possible.
```

I think?

Reply:
You're right, I wasn't sure what the difference was at the time
```suggestion
but the documentation recommends using the [notistack](https://github.com/iamhosseindhv/notistack) library to make it possible.
```